packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts

```diff
@@ -0,0 +1,279 @@
+        0,
+        importsArrayNode.getStart() + 1
+      )}\nCommonModule,${remoteEntryModule.slice(
+        importsArrayNode.getStart() + 1
+      )}`;
+      return updatedRemoteEntry;
+    } else {
+      return remoteEntryModule;
+    }
+  }
+
+  const browserModuleNode = tsquery(ast, BROWSER_MODULE_POS_AST_QUERY, {
+    visitAllChildren: true,
+  })[0];
+
+  const updatedRemoteEntryModule = `${
+    needsCommonModuleImportStatement
+      ? `import { CommonModule } from '@angular/common';\n`
+      : ``
+  }${remoteEntryModule.slice(0, browserModuleNode.getStart())}${
+    needsCommonModuleInImports ? `CommonModule` : ``
+  }${remoteEntryModule.slice(
+    browserModuleNode.getEnd() + (needsCommonModuleInImports ? 0 : 1)
+  )}`;
+
+  return updatedRemoteEntryModule;
+}
+
+function switchToES2020(tree: Tree, projects: Record<string, string>) {
+  for (const projectName of Object.keys(projects)) {
+    const { root } = readProjectConfiguration(tree, projectName);
+    let tsConfigPath = tree.exists(joinPathFragments(root, `tsconfig.app.json`))
+      ? joinPathFragments(root, `tsconfig.app.json`)
+      : joinPathFragments(root, `tsconfig.json`);
+    updateJson(tree, tsConfigPath, (json) => ({
+      ...json,
+      compilerOptions: {
+        ...json.compilerOptions,
+        target: 'ES2020',
+      },
+    }));
+  }
+}
+
+function turnMinimizeOn(tree: Tree, projects: Record<string, string>) {
+  for (const webpackPath of Object.values(projects)) {
+    let webpackConfig = tree.read(webpackPath, 'utf-8');
+    tree.write(
+      webpackPath,
+      modifyConfigToUseMinimizeOptimization(webpackConfig)
+    );
+  }
+}
+
+export function modifyConfigToUseMinimizeOptimization(
+  webpackConfig: string
+): string {
+  const OPTIMIZATION_OBJECT_AST_QUERY =
+    'PropertyAssignment:has(Identifier[name=optimization]) > ObjectLiteralExpression';
+  let ast = tsquery.ast(webpackConfig);
+  const optimizationObjectNode = tsquery(ast, OPTIMIZATION_OBJECT_AST_QUERY, {
+    visitAllChildren: true,
+  })[0];
+
+  const minimizeTrueNode = tsquery(
+    optimizationObjectNode,
+    'Identifier[name=minimize] ~ TrueKeyword',
+    { visitAllChildren: true }
+  );
+  if (minimizeTrueNode && minimizeTrueNode.length > 0) {
+    // it's already turned on
+    return webpackConfig;
+  }
+
+  const minimizeFalseNode = tsquery(
+    optimizationObjectNode,
+    'Identifier[name=minimize] ~ FalseKeyword',
+    { visitAllChildren: true }
+  );
+  if (minimizeFalseNode && minimizeFalseNode.length > 0) {
+    // it exists but it's set to false, so flip it
+    webpackConfig = `${webpackConfig.slice(
+      0,
+      minimizeFalseNode[0].getStart()
+    )}true${webpackConfig.slice(minimizeFalseNode[0].getEnd())}`;
+
+    return webpackConfig;
+  }
+
+  webpackConfig = `${webpackConfig.slice(
+    0,
+    optimizationObjectNode.getStart() + 1
+  )}\nminimize: true,${webpackConfig.slice(
+    optimizationObjectNode.getStart() + 1
+  )}`;
```

Comment:
I guess we don't need to do this? It's the default value and we don't generate it for new projects.